tests/gold_tests/continuations/openclose.test.py

```diff
@@ -51,7 +52,7 @@
         ts.Variables.port, server.Variables.Port)
 )
 
-cmd = 'curl -vs -H "host:oc.test" http://127.0.0.1:{0}'.format(ts.Variables.port)
+cmd = 'curl -H "Connection: close" -vs -H "host:oc.test" http://127.0.0.1:{0}'.format(ts.Variables.port)
```

Comment:
I'd like to make this change before landing H2 to Origin. That would ensure H2 from Client and H1 to Origin are not broken.

Reply:
Sure, I'll pull this out.

Reply:
PR #7679